.cqfd/docker/Dockerfile

```diff
@@ -12,6 +12,7 @@ ENV LC_ALL en_US.UTF-8
 RUN set -x \
     && apt-get update \
     && apt-get install -y \
+        ansible \
```

Comment:
Why don't you squash commit https://github.com/seapath/ansible/pull/621/commits/6a448643c220682222330d4283bedd83cd6d90a9 in this one ? It would be more clear in the git history

If you do so, remember to update the commit message.

Reply:
Because I have forgotten to update the branches.

I no longer use these commits.